packages/pwa-kit-extension-sdk/src/shared/utils/helpers.ts

```diff
@@ -50,3 +61,51 @@ export const kebabToLowerCamelCase = (str: string) =>
                 : word.charAt(0).toUpperCase() + word.slice(1).toLowerCase()
         )
         .join('')
+
+// Returns true if the entry passes is a ApplicationExtensionEntryArray type.
+// TODO: This looks like it could be done in a more generic way.
+const isApplicationExtensionEntryArray = (entry: ApplicationExtensionEntryArray): boolean => {
+    const [nameRef, config] = entry || []
+    return (
+        typeof nameRef === 'string' &&
+        typeof config === 'object' &&
+        !!nameRef.match(/^(?:@([^/]+)\/)?extension-(.+)$/)
+    )
+}
+
+/**
+ * Normalize and expand the extension configuration array so that it is easier to process.
+ * @param {{String, Object}[]} extensions - The extensions configuration value as defined in the PWA-Kit config.
+ * @returns {Object[]} extensions - The extensions array in object form.
+ *
+ * @example
+ * const result = expand(["store-finder", ["account-pages", {singlePage: true}], './extensions/local-extension']);
+ * console.log(result)
+ * // [["@salesforce/extension-store-finder", {}], ["@salesforce/extension-account-pages", {singlePage: true}], ["/home/project/extensions/local-extension", {}]]
+ */
+export const expand = (
```

Comment:
This `expand` is moved from the other utils file, because there is no `path` lib that is imported here in this file. I needed a file free from `path` so the `expand` can be used on the client side.